tests/Acme81/Attribute/ParameterB.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Acme81\Attribute;
+
+use Attribute;
+
+#[Attribute(Attribute::TARGET_PARAMETER)]
+class ParameterB
+{
+    public string $label;
+    public string $moreData;
+    public function __construct(
+        string $label = '',
+        string $moreData = ''
+    ) {
+        $this->label = $label;
+        $this->moreData = $moreData;
+    }
```

Comment:
We can use promoted properties here:

```suggestion
    public function __construct(
        public string $label = '',
        public string $moreData = ''
    ) {
    }
```